
Must have missed Michael's original patch that contains this change...
---
 drivers/net/atari_ethernec.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/net/atari_ethernec.c
+++ b/drivers/net/atari_ethernec.c
@@ -618,7 +618,8 @@ static int __init ne_probe1(struct net_d
 			/* start timer D, div = 1:100 */
 			mfp.tim_ct_cd = (mfp.tim_ct_cd & 0xf0) | 0x6;
 		}
-		ret = request_irq(dev->irq, ei_interrupt, 0, name, dev);
+		/* Must make this shared in case other timer ints are needed */
+		ret = request_irq(dev->irq, ei_interrupt, IRQF_SHARED, name, dev);
 		if (ret) {
 			printk(" unable to get IRQ %d (errno=%d), polling instead.\n",
 			       dev->irq, ret);
